### PR TITLE
Fix css of void status, fix handle cancel issue and change hading for…

### PR DESCRIPTION
**Pull request by nitish-beehyv**
Fix css of void status, fix handle cancel issue and change hading for unmark modal

## Summary by CodeRabbit

- **New Features**
	- Enhanced visual representation of "VOID" status in the UI with styled elements.
	- Improved dropdown item display based on user roles and filing status.
	- Dynamic modal heading and action labels based on document submission type.
	- Added dropdown fields for "PARTIES" in the "Orders" and "Hearings" tabs.

- **Bug Fixes**
	- Adjusted modal behavior for clearer feedback and improved user interactions.

### CODE REVIEW

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/AdmittedCases/AdmittedCase.js [hunk 1]
```diff
@@ -859,7 +859,12 @@ const AdmittedCases = () => {
     return {
       handleClose: handleClose,
       heading: {
-        label: "view_reason_for_voiding" === documentSubmission?.[0]?.itemType ? t("REASON_FOR_VOIDING") : t("ARE_YOU_SURE_TO_MARK_AS_VOID"),
+        label:
+          "view_reason_for_voiding" === documentSubmission?.[0]?.itemType
+            ? t("REASON_FOR_VOIDING")
+            : "unmark_void_submission" === documentSubmission?.[0]?.itemType
+            ? t("ARE_YOU_SURE_TO_UNMARK_AS_VOID")
+            : t("ARE_YOU_SURE_TO_MARK_AS_VOID"),
```
[the remark this answers is not in the document]
Comment (nitish-beehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/AdmittedCases/AdmittedCase.js:867]: checked

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/configs/UICustomizations.js [hunk 2]
```diff
@@ -1052,7 +1052,24 @@ export const UICustomizations = {
...
+                color: "#231F20",
+              }}
+            >
+              {t("VOID")}
+            </div>
+          ) : row?.status ? (
+            <CustomChip text={t(row?.status)} shade={"green"} />
+          ) : (
+            ""
+          );
```
[the remark this answers is not in the document]
Comment (nitish-beehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/configs/UICustomizations.js:1072]: checked

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/AdmittedCases/AdmittedCase.js [hunk 3]
```diff
@@ -871,9 +876,9 @@ const AdmittedCases = () => {
           ? t("MARK_VOID_CONFIRM")
           : t("MARK_AS_VOID"),
       actionCancelLabel: t("MARK_VOID_CANCEL"),
-      actionCancelOnSubmit: handleClose,
       steps: [
         {
+          actionCancelOnSubmit: handleClose,
```
[the remark this answers is not in the document]
Comment (nitish-beehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/AdmittedCases/AdmittedCase.js:881]: checked

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/AdmittedCases/AdmittedCase.js [hunk 4]
```diff
@@ -2118,7 +2123,7 @@ const AdmittedCases = () => {
           )}
         </div>
       )}
-      <div className="inbox-search-wrapper">
+      <div className="inbox-search-wrapper" style={showActionBar && !isWorkFlowFetching ? { marginBottom: "56px" } : {}}>
```
[the remark this answers is not in the document]
Comment (nitish-beehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/AdmittedCases/AdmittedCase.js:2126]: checked

**Review by TusharKatiyar2002 — APPROVED** [review]

**Pull request merged**